client/src/components/Workflow/Editor/modules/activities.ts

```diff
@@ -189,6 +198,17 @@ export function useSpecialWorkflowActivities(options: Ref<SpecialActivityOptions
             click: true,
             mutable: false,
         },
+        {
+            title: "Best Practices",
+            id: "workflow-best-practices",
+            description: "Show and test for the best practices in this workflow.",
+            tooltip: bestPracticeHover.value,
+            indicator: disconnectedCount.value,
+            icon: faMagic,
+            panel: true,
+            visible: true,
+            optional: true,
+        },
```

Comment:
I suggest moving this activity up as much as possible to make it more prominent or, at least, to let the "Save + Exit" activity be at the bottom which feels like the "last" thing you will do after you finish editing.